plugins/ngrok/credentials.go

```diff
@@ -18,8 +19,9 @@ func Credentials() schema.CredentialType {
 		DocsURL: sdk.URL("https://ngrok.com/docs/ngrok-agent/config"),
 		Fields: []schema.CredentialField{
 			{
-				Name:                fieldname.AuthToken,
-				MarkdownDescription: "Auth Token used to authenticate to ngrok.",
+				Name:                fieldname.Authtoken,
+				AlternativeNames:    []string{"Auth Token"},
+				MarkdownDescription: fmt.Sprintf("%s used to authenticate to ngrok.", fieldname.Authtoken),
```

Comment:
We're not dynamically inserting field names in descriptions in any other plugin, so I don't really see a reason to start doing it now in this plugin.

Reply:
Fair enough, modified back